vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -82,7 +82,7 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.27.8", # for k8s 1.27.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.28.0", # for k8s 1.28.x
     # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-    "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.14",
+    "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.15",
```

Comment:
@pfrcks The image is very large and it needs a long time to download it if it is not cached in AKS Windows images. It is better to keep the old image version if the disk space in the vhd is enough. It should be better to cache the new image version before Patch Tuesday when AKS builds monthly AKS Windows images.

Reply:
@AbelHu we started cleaning up older images based on feedback from VHD team to reduce the amount of space ciprod images take on the VHD considering ciprod win images are 2-3Gi